linchpin/provision/roles/openshift/files/schema.json

```diff
@@ -10,8 +10,11 @@
                     "allowed": ["openshift_inline", "openshift_external"]
                 },
                 "name": { "type": "string", "required": true },
-                "data": { "required": false },
-                "filename": { "required": false }
+                "force": {"type": "boolean", "required": false },
+                "kind": {"type": "string", "required": false },
```

Comment:
kind , required:true

Reply:
@skatlapa As mentioned in previous comment , kind is not needed when we use the definition, src attributes.